### PR TITLE
Create segm8 package

### PR DESCRIPTION
**Pull request by AlexErAmp**

### CODE REVIEW

File: segm8/segm8.py [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+            omitted, defines a single module.
+        """
+        self._pin_CE = pin_CE
+        self._device_count = device_count
+        # Set the internal buffer to the "dark" state.
+        self._data = [font.FONT[" "]] * self._device_count
+        # Create an SPI object on the "0" SPI bus and connected to
+        # "pin_CE".
+        self._spi = self._spi_init(0, pin_CE)
+
```
Comment (yurybotov) [inline segm8/segm8.py:43]: Shouldn't you call _send_data at the end of the constructor to clear the indicators?

File: segm8/output_format.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+number systems used by SegM8 objects."""
+
+
+class Align:
+    """Constants describing the alignment method."""
+    RIGHT = 0
+    LEFT = 1
+
+
+class NumberSystem:
```
Comment (yurybotov) [inline segm8/output_format.py:11]: IMHO "Radix"  is ​​shorter and quite unambiguously means the same thing. At your discretion.

File: segm8/segm8.py [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+        pin_CE: int
+            The chip enable pin.
+
+        Returns:
+        --------
+        SpiDev object
+        """
+        spi = spidev.SpiDev()
+        spi.open(bus, pin_CE)
+        spi.max_speed_hz = 62500000
```
Comment (yurybotov) [inline segm8/segm8.py:240]: "spi.max_speed_hz = 62500000"
I'm not sure if these modules will be able to work stably at this SPI frequency. Especially if the chain is long. I was only able to test them up to 1 MHz

**Review by yurybotov — COMMENTED** [review]
Looking good. See comments. 
I couldn't test on PI, I don't have it at home.

File: segm8/segm8.py (hunk 1 above)
Comment (AlexErAmp) [reply segm8/segm8.py:43]: Yes thanks, missed that. Fixed in the next commit.

**Review by nkrkv — APPROVED** [review]

File: segm8/output_format.py (hunk 2 above)
Comment (AlexErAmp) [reply segm8/output_format.py:11]: To be honest, this is the first time I've heard this word from you :) I asked three people in the office: one knows, two do not.
Perhaps I'll leave it that way.

File: segm8/segm8.py (hunk 3 above)
Comment (AlexErAmp) [reply segm8/segm8.py:240]: I was unable to properly display data at frequencies lower than this on the RPi.

**Review by yurybotov — APPROVED** [review]
Ok

**Pull request merged**